arm-documentdb/2015-04-08/documentdb.json

```diff
@@ -0,0 +1,767 @@
+      },
+      "required": [
+        "consistencyPolicy",
+        "locations"
+      ]
+    },
+    "DatabaseAccountCreateUpdateParameters": {
+      "description": "DocumentDB Database Account Create and Update parameters.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DatabaseAccountCreateUpdateProperties"
+        },
+        "databaseAccountOfferType": {
+          "$ref": "#/definitions/DatabaseAccountOfferType"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "required": [
+        "databaseAccountOfferType"
+      ]
+    },
+    "DatabaseAccountPatchTagsParameters": {
+      "description": "Database Account parameters for patching tags.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "$ref": "#/definitions/Tags"
+        }
+      },
+      "required": [
+        "tags"
+      ]
+    },
+    "DatabaseAccountListReadOnlyKeysResult": {
+      "description": "The read-only access keys for the given database account.",
+      "properties": {
+        "primaryReadonlyMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the primary read-only key."
+        },
+        "secondaryReadonlyMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the secondary read-only key."
+        }
+      }
+    },
+    "DatabaseAccountListKeysResult": {
+      "description": "The access keys for the given database account.",
+      "properties": {
+        "primaryMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the primary read-write key."
+        },
+        "secondaryMasterKey": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Base 64 encoded value of the secondary read-write key."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DatabaseAccountListReadOnlyKeysResult"
+        }
+      }
+    },
+    "DatabaseAccountRegenerateKeyParameters": {
+      "type": "object",
+      "description": "Parameters to regenerate the keys within the database account.",
+      "properties": {
+        "keyKind": {
+          "type": "string",
+          "description": "The access key to regenerate.",
+          "enum": [
+            "primary",
+            "secondary",
+            "primaryReadonly",
+            "secondaryReadonly"
+          ],
+          "x-ms-enum": {
+            "name": "KeyKind",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "DatabaseAccountOfferType": {
+      "description": "The offer type for the DocumentDB database account.",
+      "type": "string",
+      "enum": [
+        "Standard"
+      ]
```

Comment:
This value is to be deprecated so no more values are to be created in the future